lib/lambda-cfn.js

```diff
@@ -1002,36 +977,29 @@ function outputs(options) {
   return output;
 }
 
-function streambotEnv(options) {
-  if (!options.name)
-    throw new Error('name property required for streambotEnv');
+function envVariableParser(options) {
 
-    var env = {
-      "Type": "Custom::StreambotEnv",
-      "Properties": {
-        "ServiceToken": {
-          "Ref": "StreambotEnv"
-        },
-        "FunctionName": {
-          "Ref": options.name
-        }
-      }
-    };
+    var p = !options.parameters ? {} :
+        JSON.parse(JSON.stringify(parameters(options)));
 
-  var p = !options.parameters ? {} :
-      JSON.parse(JSON.stringify(parameters(options)));
+    for (var k in p) {
```

Comment:
What's `k` and what's `p`? Longer, descriptive variable names would help for ease of reading